modin/pandas/test/test_series.py

```diff
@@ -166,35 +170,34 @@ def inter_df_math_helper_one_side(
     ]:
         return
 
-    try:
-        pandas_result = pandas_attr(pandas_series)
-    except Exception as err:
-        with pytest.raises(type(err)):
-            repr(modin_attr(modin_series))  # repr to force materialization
-    else:
-        modin_result = modin_attr(modin_series)
-        df_equals(modin_result, pandas_result, **comparator_kwargs)
+    eval_general(
+        modin_series,
+        pandas_series,
+        lambda df: pandas_attr(df) if isinstance(pandas.Series) else modin_attr(df),
```

Comment:
```suggestion
        lambda df: (pandas_attr if isinstance(df, pandas.Series) else modin_attr)(df),
```
how does this pass any tests with this typo btw? 🤣 (missing argument to `isinstance()`)

Reply:
both pandas and modin threw the same exception and the test passed. Still, this behavior of `eval_general` function is very dangerous.

Reply:
good catch!